guide/mythic_plus/mythic_plus.md

```diff
@@ -72,7 +83,12 @@ In coordinated and varied group compositions, you may also consider {{ site.data
+* [Oros Coldheart](https://shadowlands.wowhead.com/npc=179892/oros-coldheart) offers [Regenerative Fungus](https://shadowlands.wowhead.com/spell=357814/regenerative-fungus), [Vial of Desperation](https://shadowlands.wowhead.com/spell=357825/vial-of-desperation), and [Handbook of Uncivil Etiquette](https://shadowlands.wowhead.com/spell=357834/handbook-of-uncivil-etiquette). The only damage-increasing option available is [Handbook of Uncivil Etiquette](https://shadowlands.wowhead.com/spell=357834/handbook-of-uncivil-etiquette) which encourages you to interrupt as often as possible! A reasonable alternative is [Regenerative Fungus](https://shadowlands.wowhead.com/spell=357814/regenerative-fungus).
+* [Soggodon the Breaker](https://shadowlands.wowhead.com/npc=179891/soggodon-the-breaker) offers [The Stone Ward](https://shadowlands.wowhead.com/spell=357524/the-stone-ward), [Dripping Fang](https://shadowlands.wowhead.com/spell=356828/dripping-fang), and [Tiny Dancing Shoes](https://shadowlands.wowhead.com/spell=356827/tiny-dancing-shoes). The default choice is [The Stone Ward](https://shadowlands.wowhead.com/spell=357524/the-stone-ward) for its strong effect.
+* [Executioner Varruth](https://shadowlands.wowhead.com/npc=179890/executioner-varruth) offers [Champion's Brand](https://shadowlands.wowhead.com/spell=357575/champions-brand), [Dagger of Necrotic Wounding](https://shadowlands.wowhead.com/spell=357609/dagger-of-necrotic-wounding), and [Volcanic Plumage](https://shadowlands.wowhead.com/spell=357706/volcanic-plumage). There is more flexibility in this choice than the others; [Champion's Brand](https://shadowlands.wowhead.com/spell=357575/champions-brand) is a flexible default choice, [Dagger of Necrotic Wounding](https://shadowlands.wowhead.com/spell=357609/dagger-of-necrotic-wounding) is a Single Target orientated effect which makes it good for bosses and less-good at trash packs, [Volcanic Plumage](https://shadowlands.wowhead.com/spell=357706/volcanic-plumage) is an AoE orientated effect which provides an additional element of crowd-control via the knock-up effect but is less-good for boss encounters. As always make appropriate decisions for the level, dungeon, and group you're with. **Your choice here will not define your success or failure in a particular key**
```

Comment:
"As always make appro[...]" reads like a conclusion/reminder not specific to this bullet point. could it be just the conclusion after this list?